[MWAN] Connector app based steering (#14933)
cloudflare/cloudflare-docs: content/magic-wan/configuration/connector/network-options/application-based-policies/breakout-traffic.md

pcx_content_type: how-to
title: Breakout traffic
meta:
    description: Breakout traffic allows you to define which applications should bypass Cloudflare’s security filtering.
---

# Breakout traffic

Breakout traffic allows you to define which applications should bypass Cloudflare’s security filtering, and go directly to the Internet. It works via DNS requests inspection. This means that if your network is caching DNS requests, Breakout traffic will only take effect after you cache entries expire, and your client issues a new DNS request that the Magic WAN Connector can detect. This can take several minutes.

Breakout traffic will not work for applications that use DNS-over-HTTPs.

{{<Aside type="note">}}

> Cloudflare’s best practice recommendation is to route all traffic through our global network for comprehensive security filtering and access controls, but there may be specific cases where you want a subset of traffic to bypass Cloudflare’s security filtering and route directly to the Internet. You can scope this breakout traffic to specific applications from the Cloudflare dashboard.

Reviewer: ```suggestion
We recommend routing all traffic through our global network for comprehensive security filtering and access controls, but there may be specific cases where you want a subset of traffic to bypass Cloudflare’s security filtering and route directly to the Internet. You can scope this breakout traffic to specific applications from the Cloudflare dashboard.
```
